src/builtins/danger_zone.rs

```diff
@@ -125,3 +125,20 @@ impl RuleSet for ForkAndExec {
         "ForkAndExec"
     }
 }
+
```

Comment:
I/we should probably change the doc comment in ForkAndExec as well as here to first describe what this actually allows, then put the rest into a `# Security considerations` section.

I also don't think that the description is entirely accurate - I believe you can send file descriptors across unix domain sockets so presumably you can send the pipe's fd along those as well, and the other process can be anything.